vhd/packer/install-dependencies.sh

```diff
@@ -224,6 +224,7 @@ for TILLER_VERSION in ${TILLER_VERSIONS}; do
 done
 
 CLUSTER_AUTOSCALER_VERSIONS="
+1.18.0
```

Comment:
changes in this file should go to https://github.com/Azure/AgentBaker/blob/master/vhdbuilder/packer/install-dependencies.sh as AKS build its own VHD